### PR TITLE
AP_OSD: new osd items

### PR DESCRIPTION
**Pull request by sh83**
New osd items:
 - added direction to the groundspeed
 - compass item
 - wind speed/direction item
 - airspeed item
 - climbrate item

WIP because this PR require testing and checking

### CODE REVIEW

File: libraries/AP_OSD/AP_OSD_Screen.cpp [hunk 1]
```diff
@@ -101,11 +101,27 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
...
+    //@Path: AP_OSD_Setting.cpp
+    AP_SUBGROUPINFO(compass, "COMPASS", 17, AP_OSD_Screen, AP_OSD_Setting),
+
+    //@Group: WIND
+    //@Path: AP_OSD_Setting.cpp
+    AP_SUBGROUPINFO(wind, "WIND", 18, AP_OSD_Screen, AP_OSD_Setting),
+
+    //@Group: ASPEED
+    //@Path: AP_OSD_Setting.cpp
+    AP_SUBGROUPINFO(wind, "ASPEED", 19, AP_OSD_Screen, AP_OSD_Setting),
```
Comment (tridge) [inline libraries/AP_OSD/AP_OSD_Screen.cpp]: wrong variable (should be aspeed)

File: libraries/AP_OSD/AP_OSD_Screen.cpp [hunk 2]
```diff
@@ -101,11 +101,27 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
...
+
+    //@Group: VSPEED
+    //@Path: AP_OSD_Setting.cpp
+    AP_SUBGROUPINFO(wind, "VSPEED", 20, AP_OSD_Screen, AP_OSD_Setting),
```
Comment (tridge) [inline libraries/AP_OSD/AP_OSD_Screen.cpp]: wrong variable

**Comment by Kelly-Foster** [comment]
same error on line 123 as line 119.  Cut & paste of wind should be aspeed and vspeed instead.

File: libraries/AP_OSD/AP_OSD_Screen.cpp [hunk 3]
```diff
@@ -324,6 +360,72 @@ void AP_OSD_Screen::draw_throttle(uint8_t x, uint8_t y)
     backend->write(x, y, false, "%3d%c", gcs().get_hud_throttle(), SYM_PCNT);
 }
 
+//Thanks to betaflight/inav for simple and clean compass visual design
+void AP_OSD_Screen::draw_compass(uint8_t x, uint8_t y)
+{
+    const int8_t total_sectors = 16;
+    char compass_circle[total_sectors] = {
```
[the remark this answers is not in the document]
Comment (sh83) [reply libraries/AP_OSD/AP_OSD_Screen.cpp]: Thank you. Fixed now.

File: libraries/AP_OSD/AP_OSD_Screen.cpp (hunk 1 above)
Comment (sh83) [reply libraries/AP_OSD/AP_OSD_Screen.cpp]: Fixed now.

File: libraries/AP_OSD/AP_OSD_Screen.cpp (hunk 2 above)
Comment (sh83) [reply libraries/AP_OSD/AP_OSD_Screen.cpp]: Fixed now.

**Comment by sh83** [comment]
@Kelly-Foster Thank you for testing this. I have fixed flashed wind symbol and var name bugs.

**Comment by Kelly-Foster** [comment]
Tested commit 610f3a6.  All good.

**Pull request merged**